rustls/src/msgs/alert.rs

```diff
@@ -12,9 +12,9 @@ pub struct AlertMessagePayload {
 }
 
 impl Codec for AlertMessagePayload {
-    fn encode(&self, bytes: &mut Vec<u8>) {
-        self.level.encode(bytes);
-        self.description.encode(bytes);
+    fn try_encode<B: TryPushBytes>(&self, bytes: &mut B) -> Result<(), B::Error> {
```

Comment:
I think we should avoid renaming the method in the same commit (or, in the current state, PR) as change the type, since this creates a bunch of otherwise unrelated churn.

Reply:
I think I don't follow. `Codec::encode` is still used as it was before, it's signature changed but it is backwards compatible with the old one.

Reply:
Ahh, sorry, yes, I see what you did now.